packages/cosmos_ui_components/lib/components/cosmos_image_button.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:cosmos_ui_components/cosmos_ui_components.dart';
+import 'package:flutter/material.dart';
+
+class CosmosImageButton extends StatelessWidget {
+  final String assetUrl;
+  final VoidCallback onPressed;
+
+  const CosmosImageButton({
+    Key? key,
+    required this.assetUrl,
+    required this.onPressed,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: () => onPressed(),
+      child: SizedBox(
+        height: 24,
+        width: 24,
```

Comment:
we could parametrize the width and height as I'm pretty sure we'll have many different sizes for the button

Reply:
Updated